lang/en/docs/tutorials/python-ml/predict-with-classification.md

```diff
@@ -0,0 +1,94 @@
+# Machine Learning: Predict With a Random Forest for Classification
+
+This tutorial demonstrates how to perform predictions using a Random Forest [^1] trained for classification via
+Scikit-Learn. [^2]
+
+!!! warning "Pre-Requisites"
+    In order to perform this tutorial, the [Predict with Classification](predict-with-classification.md) tutorial must
+    be completed.
+
+## 1. Acquire Data
+
+The data we use in this example comes from the QSAR group's biodegredation database, as hosted on Kaggle. [^3]
+
+The dataset consists of 41 unique descriptors of each molecule, and the goal of the problem is to predict whether the
+molecule is biodegredable or not.
+
+The dataset can be found on Kaggle, [here](https://www.kaggle.com/muhammetvarl/qsarbiodegradation).
```

Comment:
Is this a permanent link or should we have a copy of this dataset somewhere (Google Drive?) in case the link becomes unavailable? Same for the clustering tutorial.

Reply:
Kaggle is generally good about keeping datasets around a long time. We could also host it in the docs (with a link back to the original copy) as we did for the clustering example.